examples/flutter_gallery/lib/demo/calculator/home.dart

```diff
@@ -252,9 +252,7 @@ class CalcKey extends StatelessWidget {
         child: Center(
           child: Text(
             text,
-            style: TextStyle(
-              fontSize: (orientation == Orientation.portrait) ? 32.0 : 24.0
-            ),
+            style: TextStyle(fontSize: (orientation == Orientation.portrait) ? 32.0 : 24.0),
```

Comment:
the original probably needs a trailing comma

Reply:
That's a common problem I saw. There's no formatting consistency for a given Widget. I'd like to make that better with a lint (but it is another problem)